README.md

```diff
@@ -27,7 +25,7 @@ go get golang.org/x/net/context
 Put the package under your project folder and add the following in import:
 
 ```golang
-import basistheory "github.com/Basis-Theory/basistheory-go/v3"
+import basistheory "github.com/GIT_USER_ID/GIT_REPO_ID"
```

Comment:
need to do undo a few changes in this file. we should probably create a README template for the README to avoid doing this every tme